examples/problem_decomposition/ifci_mifno.ipynb

```diff
@@ -0,0 +1,554 @@
+    "\n",
+    "This notebook assumes that you already have installed Tangelo in your Python environment, or have updated your Python path so that the imports can be resolved. If not, executing the cell below installs the minimal requirements for this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Installation of tangelo if not already installed.\n",
+    "try:\n",
+    "    import tangelo\n",
+    "except ModuleNotFoundError:\n",
+    "    !pip install pyscf\n",
+    "    !pip install git+https://github.com/goodchemistryco/Tangelo.git@develop --quiet\n",
+    "\n",
+    "# Download the data folder at https://github.com/goodchemistryco/Tangelo/branches/develop/tangelo/problem_decomposition/tests/incremental/data/BeH2_CCPVDZ_MIFNO_HBCI\n",
+    "import os\n",
+    "data_folder = \"BeH2_CCPVDZ_MIFNO_HBCI\"\n",
+    "\n",
+    "if not os.path.isdir(data_folder):\n",
+    "    !sudo apt install subversion\n",
+    "    !svn export https://github.com/goodchemistryco/Tangelo/branches/develop/tangelo/problem_decomposition/tests/incremental/data/{data_folder}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use case\n",
+    "\n",
+    "Our use case here is the beryllium hydride (BeH$_2$) system defined below, using the `cc-pvdz` basis, chosen for simplicity."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xyz = \"\"\"\n",
+    "Be  0.0000000   0.0000000    0.0000000\n",
+    "H   0.0000000   0.0000000    1.3300000\n",
+    "H   0.0000000   0.0000000   -1.3300000\n",
+    "\"\"\"\n",
+    "basis = \"cc-pvdz\"\n",
+    "charge = 0 \n",
+    "spin = 0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What are iFCI and MI-FNO ?\n",
+    "\n",
+    "The method of increment (MI) expresses the electron correlation energy of a molecular system as a truncated many-body expansion in terms of orbitals, atoms, molecules, or fragments. The electron correlation of the system is expanded in terms of occupied orbitals, and MI is employed to systematically reduce the occupied orbital space. At the same time, the virtual orbital space is reduced based on orbital lozalization and the one-particle density matrix from the second-order many-body perturbation theory. In this way, the methods referred to as [iFCI](https://doi.org/10.1063/1.4977727) or [MI-FNO](https://doi.org/10.1063/5.0054647) are available for the systematic reduction of both the occupied space and the virtual space in quantum chemistry simulations. The two methods share similarities, but they are distinguished by a few key differences: the approach to truncating the virtual orbital space and the inclusion of a correction factor for each MI-FNO fragment (which is not required for iFCI fragments).\n",
```

Comment:
The method of increment (MI) expresses the electron correlation energy of a molecular system as a truncated many-body expansion in terms of orbitals, atoms, molecules, or fragments. The electron correlation of the system is expanded in terms of occupied orbitals, and MI is employed to systematically reduce the occupied orbital space. Simultaneously, the virtual orbital space is reduced based on orbital localization and the one-particle density matrix from the second-order many-body perturbation theory. Following this approach, the methods referred to as [iFCI](https://doi.org/10.1063/1.4977727) and [MI-FNO](https://doi.org/10.1063/5.0054647) are available for the systematic reduction of both the occupied space and the virtual space in quantum chemistry simulations. Although the two methods share similarities, they exhibit a few key differences, including the approach used to truncate the virtual orbital space (FNO vs SNO) and the inclusion of a correction factor for each MI-FNO fragment (which is not required for iFCI fragments).